ztp/gitops-subscriptions/argocd/WorkloadPartitioning.md

```diff
@@ -0,0 +1,38 @@
+# Workload Partitioning Feature
+
+Workload partitioning is the feature that allows the platform to dedicate specific CPUSets to customer workloads and platform workloads (~2 cores for platform and rest for customers). This feature **must** be activated at install time of a cluster, once active that cluster is locked into functioning with this feature on.
```

Comment:
```suggestion
Workload partitioning allows the platform to dedicate specific CPUSets to customer workloads and platform workloads (~2 cores for platform and the rest for customers). This feature **must** be activated at install time of a cluster, once active that cluster is locked into functioning with this feature on.
```

I don't remember if this "must be activated at install time" is still true, though?  We assumed it was, but maybe it's not?  @imiller0 do you know more about this?

Reply:
My recollection is that workload partitioning must be set during initial install to ensure that pods are correctly mutated during admission. So long as it is enabled at initial install the cpuset can be modified at day-N. @eggfoobar has been working on this most recently and may know better if it is still required to be enabled at cluster install.

Reply:
Yep Egli actually [suggested](https://github.com/openshift-kni/cnf-features-deploy/pull/1376#discussion_r1065922360) this change